source/code-snippets/usage-examples/deleteOne.js

```diff
@@ -0,0 +1,41 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // Query for a movie that has a title of type string
+    const query = { title: { $type: "string" } };
+
+    const options = {
```

Comment:
Let's actually get rid of these options here since they can be specified on the connection, database, and collection level, and should be specified there.